### DISCUSSION TITLE
Set-PnPListItem not setting the "Modified" value in the first run when using the batch

### DISCUSSION THREAD

**Discussion by AshokChava** [pnp/powershell#4470] [Q&A]
We are setting following properries for the assets in onedrive/sharepoint
    $PropValues = @{
    "Title" = $item.Title
    "Author" = $item.Author
    "Editor" = $item.Editor
    "Created" = $item.Created
    "Modified" = $item.Modified
    "Description0" = $item.Description0
    "Keywords0" = $item.Keywords0
    "Migration_Comments0" = $item.Migration_Comments0
    }
    Using the command "Set-PnPListItem -List $ListName -Identity $itemID -Values $PropValues -batch $batch -UpdateType UpdateOverwriteVersion -Force | Out-Null"
    When we run this cript for the first time except for "Modified" all the other values are set fine, to fix the "Modified" field value we have to run the script again.

**Comment by mouzzampk** [comment]
I think this is same as #4463 where I'm having same issue.
